src-cljs/atom_parinfer/core.cljs

```diff
@@ -273,6 +273,42 @@
 
 (add-watch *editor-states :toggle-status-classes toggle-status-classes!)
 
+;;------------------------------------------------------------------------------
+;; Error Markers
+;;------------------------------------------------------------------------------
+
+(def error-marker-class "parinfer-error-marker")
```

Comment:
sorry, I've been trying to match your idioms in this repo, but I think this is the first user-customizable class and I thought a hash might be unnecessary here and maybe jarring to users.